README.md

```diff
@@ -47,6 +47,88 @@ make notebook
 
 and selecting the pysyft kernel
 
+## Use the Docker image
+
+Instead of installing all the dependencies on your computer, you can run a notebook server (which comes with Pysyft installed) using [Docker](https://www.docker.com/). All you will have to do is start the container like this:
+
+```bash
+$ docker container run youben/pysyft:miniconda
+[I 23:41:51.429 NotebookApp] Writing notebook server cookie secret to /root/.local/share/jupyter/runtime/notebook_cookie_secret
+[I 23:41:52.865 NotebookApp] Serving notebooks from local directory: /workspace
+[I 23:41:52.865 NotebookApp] The Jupyter Notebook is running at:
+[I 23:41:52.865 NotebookApp] http://172.17.0.2:8888/?token=e0bb2ed0c185ae35b02f512ff293ed97fa0d2300b2ff73b3
+[I 23:41:52.866 NotebookApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
+[W 23:41:52.870 NotebookApp] No web browser found: could not locate runnable browser.
+[C 23:41:52.870 NotebookApp]
+
+    To access the notebook, open this file in a browser:
+        file:///root/.local/share/jupyter/runtime/nbserver-10-open.html
+    Or copy and paste one of these URLs:
+        http://172.17.0.2:8888/?token=e0bb2ed0c185ae35b02f512ff293ed97fa0d2300b2ff73b3
+
+```
+
+You can use the provided link to access the jupyter notebook (the link is only accessible from your local machine).
+
+
+You could also build the image on your own and run it locally:
+
+```bash
+$ cd docker-image
+$ docker image build -t pysyft-notebook .
+Sending build context to Docker daemon    129kB
```

Comment:
Can you remove the lines of output from docker for brevity

Reply:
done